ts/screens/wallet/payment/TransactionErrorScreen.tsx

```diff
@@ -48,277 +71,272 @@ type Props = OwnProps &
-  },
-
-  errorMessageSubtitle: {
-    textAlign: "center"
-  },
+const baseIconPath = "../../../../img/";
 
-  paddedLR: {
-    paddingLeft: customVariables.contentPadding,
-    paddingRight: customVariables.contentPadding
-  }
-});
-
-const MAX_MINUTES_FOR_PAYMENT_DELETION = 15;
+const imageMapping: Record<ErrorTypes, ImageSourcePropType> = {
+  DATA: require(baseIconPath + "pictograms/doubt.png"),
+  DUPLICATED: require(baseIconPath + "pictograms/fireworks.png"),
+  EC: require(baseIconPath + "wallet/errors/payment-unavailable-icon.png"),
+  EXPIRED: require(baseIconPath + "servicesStatus/error-detail-icon.png"),
+  ONGOING: require(baseIconPath + "pictograms/hourglass.png"),
+  REVOKED: require(baseIconPath + "servicesStatus/error-detail-icon.png"),
+  UNCOVERED: require(baseIconPath + "/wallet/errors/generic-error-icon.png"),
+  TECHNICAL: require(baseIconPath + "servicesStatus/error-detail-icon.png")
+};
 
-/**
- * Print the icon according to current error status
- * @param error
- */
-export const renderErrorTransactionIcon = (
-  error: NavigationParams["error"]
+// Save the rptId as attribute and open the Instabug chat.
+const requestAssistanceForPaymentFailure = (
+  rptId: RptId,
+  payment?: PaymentHistory
 ) => {
-  const baseIconPath = "../../../../img/wallet/errors/";
-  switch (error.toUndefined()) {
-    case "PAYMENT_DUPLICATED":
-      return require(baseIconPath + "payment-duplicated-icon.png");
-    case "INVALID_AMOUNT":
-      return require(baseIconPath + "invalid-amount-icon.png");
-    case "PAYMENT_ONGOING":
-      return require(baseIconPath + "payment-ongoing-icon.png");
-    case "PAYMENT_EXPIRED":
-      return require(baseIconPath + "payment-expired-icon.png");
-    case "PAYMENT_UNAVAILABLE":
-      return require(baseIconPath + "payment-unavailable-icon.png");
-    case "PAYMENT_UNKNOWN":
-      return require(baseIconPath + "payment-unknown-icon.png");
-    case "DOMAIN_UNKNOWN":
-      return require(baseIconPath + "domain-unknown-icon.png");
-    case "PAYMENT_ID_TIMEOUT":
-      return require(baseIconPath + "missing-payment-id-icon.png");
-    default:
-      return require(baseIconPath + "generic-error-icon.png");
+  Instabug.appendTags([paymentInstabugTag]);
+  setInstabugUserAttribute(
+    "blockedPaymentRptId",
+    RptIdFromString.encode(rptId)
+  );
+  if (payment) {
+    instabugLog(
+      getPaymentHistoryDetails(payment),
+      TypeLogs.INFO,
+      paymentInstabugTag
+    );
   }
+  openInstabugQuestionReport();
+};
+
+type ScreenUIContents = {
+  image: ImageSourcePropType;
+  title: string;
+  subtitle?: React.ReactNode;
+  footerButtons?: ComponentProps<typeof FooterStackButton>["buttons"];
 };
 
+const ErrorCodeCopyComponent = ({
+  error
+}: {
+  error: keyof typeof Detail_v2Enum;
+}): React.ReactElement => (
+  <View testID={"error-code-copy-component"}>
+    <H4 weight={"Regular"}>{I18n.t("wallet.errors.assistanceLabel")}</H4>
+    <H4 weight={"Bold"} testID={"error-code"}>
```

Comment:
This PR LGTM
This is the only suggestion that I already tested (if you want to commit it directly)
```suggestion
    <H4 weight={"Bold"} testID={"error-code"}  style={{textAlign: "center"}}>
```

| now  | suggestion |
| ------------- | ------------- |
![now](https://user-images.githubusercontent.com/822471/134874820-b6dc3f60-30e8-4c19-889f-f42f48c745d9.png) | ![post](https://user-images.githubusercontent.com/822471/134874834-b45b60ad-a698-4d48-b5e8-c874be531c73.png)